test/test_linalg.py

```diff
@@ -5866,6 +5866,34 @@ def _gen_pair(m, k, n):
                                r"Expected result.size\(0\) to be 17 but got 16",
                                lambda: torch._int_mm(genf_int(17, 8), genf_int(8, 32), out=genf_int(16, 31).int()))
 
+    @onlyCPU
```

Comment:
we should expand the existing test case `test__int_mm` instead of creating a new one for cpu.

Reply:
The test case for CUDA has many restrictions and checks because the CUDA implementation has many limitations of shape and CUDA version, etc. However, the CPU implementation does not have those limitations. So, it will be much easier to separate the tests for CUDA and CPU. Do you think it's OK? Thanks!

Reply:
Can you still extend the CUDA case, and add some CPU-only shapes to the test?

Reply:
Hi @lezcano Sorry I did not notice this comment. Do I still need to combine the CPU and CUDA test cases?